src/cljs/athens/views/blocks/textarea_keydown.cljs

```diff
@@ -578,7 +578,10 @@
       (= selection "") (let [new-str (str head key close-pair tail)
                              new-idx (inc start)]
                          (swap! state assoc :string/local new-str)
-                         (set! (.-value target) new-str)
+                         (.. js/document (execCommand
```

Comment:
Thanks for the hint -- I'll add it and look for alternatives that are not deprecated.

Reply:
> Since you are already editing this fn
> can you change https://github.com/athensresearch/athens/pull/1194/files#R572 to 575
> to
> (every? #(= % key lookbehind) ["}" ....]) ?

I suppose you mean `some`?

Reply:
`some` is the right one. thanks.